<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>390</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>apply</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cov_type">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Identity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edge ↕</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edge ↔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Emboss</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gradient ↕</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gradient ↔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sharpen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Smooth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobel ↕</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobel ↔</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="nomal">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter normalization    (MINMAX)</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="originImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="filterImage">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="useImage">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>use filter image</string>
    </property>
   </widget>
   <widget class="QToolButton" name="filedialog">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>origin image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>430</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>filter image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>kernel</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>(-99~99)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
